library/general/src/lib/yast2/popup.rb

```diff
@@ -23,6 +23,10 @@ class << self
       RICHTEXT_WIDTH = 60
       RICHTEXT_HEIGHT = 10
 
+      # Minimum width for auto wrapped labels. It is computed that max line or this
```

Comment:
NP: I am not sure if this comment is clear. If I got it correctly, what about this?

~~~
# Minimum width for auto wrapped labels. This value is used when
# the label has longer lines.
~~~